### DISCUSSION TITLE
WebGIS6 - Desktop-Layout

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#185] [Q&A]
Hallo Jürgen!

Im Desktop-Layout im WebGIS6 (6.24.2706) gibt es die Möglichkeit die Größe vom TOC zu ändern. Bei Mouse Over wird die Leiste breiter und ganz oben in der Leiste erscheint ein schwarzer Pfeil (nach unten gedrehtes Dreieck - siehe Screenshot).

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/25852d8d-2357-4e09-8856-f05035234538)

Wenn ich auf diesen Pfeil klicke verschwindet die Karte, die Werkzeugleiste und die Möglichkeit den TOC in der Größe zu ändern. Nur mit einem Browser-Refresh kann ich die das Layout wiederherstellen. 

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/9bc08378-56d0-48bc-a66a-de41410db6c4)

Könntest du dir das bitte mal anschauen. 
Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Stimmt, das war noch was... hab ich auf meiner Agenda.
Wundert mich, dass das bis jetzt keinem Aufgefallen ist. Setzt ich natürlich um

**Reply by gis-eni** [reply]
Es gibt einen neuen Build 6.24.2801.
Dort ist der "Pfeil" nicht mehr mit dabei.

Den Pfeil gibt es derweil nur noch zum minimieren der Ergebnis Tabelle

**Comment by Andreas-Lampert** [comment]
Vielen Dank Jürgen! Passt in Version 6.24.2802!
